assets/js/modules/search-console/components/common/ModuleHeader.js

```diff
@@ -0,0 +1,83 @@
+/**
+ * ModuleHeader component.
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+
+import Link from '../../../../components/Link';
+
+function ModuleHeader( { title, ctaLabel, ctaLink } ) {
```

Comment:
See above, this is a parallel effort to what's established in #3015 - apologies for not having flagged/noticed this as overlap. We can remove this component again and rely on the approach in #3015 which is quite simpler.